packages/common/src/services/audius-backend/wAudio.ts

```diff
@@ -0,0 +1,24 @@
+import BN from 'bn.js'
+
+const MIN_WAUDIO_AMOUNT = new BN('10000000000') // 10^10
+
+/**
+ * Converts Wei Audio (BN) to wAudio (BN).
+ * wAudio has only 8 digits of precision vs. 18 in Wei Audio.
+ * The amount must be >= 10^10 and have no remainder when divided by 10^10.
+ */
+export const wAudioFromWeiAudio = (amount: BN) => {
```

Comment:
I assume @rickyrombo is going to suggest that we should be using FixedDecimal and not writing new conversion utilities for this?